### PR TITLE
MCKIN-22396 Fix samesite cookie issue for Scorm Shell login flow

### PR DESCRIPTION
**Pull request by moeez96**
Ticket: https://edx-wiki.atlassian.net/browse/MCKIN-22396

In Scorm shell login flow, the platform is loaded in an iframe from a cross origin ancestor. Hence we need the sessionid and csrftoken cookies to have samesite='None' and secure=True (after chrome update).

Commits:
https://github.com/edx/edx-platform/commit/7b3525278ebe3efbd7138a65343aff609de41838
https://github.com/edx/edx-platform/commit/bb85420e914978dc8c53d8bb180afc2a4b0d580c

Related PR without cherry-picked commits: https://github.com/edx-solutions/edx-platform/pull/1876

### CODE REVIEW

File: openedx/core/djangoapps/cors_csrf/tests/test_middleware.py [hunk 1]
```diff
@@ -269,7 +269,9 @@ def _assert_cookie_sent(self, response, is_set):
                 domain=self.COOKIE_DOMAIN
             )
             self.assertIn(expected, cookie_header)
-            self.assertIn('Max-Age=31449600; Path=/; secure', cookie_header)
+            self.assertIn('Max-Age=31449600;', cookie_header)
+            self.assertIn('Path=/;', cookie_header)
+            self.assertIn('secure', cookie_header.lower())
```
Comment (ihtram) [inline openedx/core/djangoapps/cors_csrf/tests/test_middleware.py:274]: @moeez96 Please replace these three lines with a single line referenced from upstream 
https://github.com/edx/edx-platform/blob/8c43855cfe6660a484e80c0b6540200bd263f52a/openedx/core/djangoapps/cors_csrf/tests/test_middleware.py#L275

**Review by ihtram — COMMENTED** [review]
Please squash the last two commits, added by you, to a single commit.

**Review by ihtram — APPROVED** [review]
👍

File: openedx/core/djangoapps/cors_csrf/tests/test_middleware.py (hunk 1 above)
Comment (moeez96) [reply openedx/core/djangoapps/cors_csrf/tests/test_middleware.py:274]: We have seen that tests fail in that case https://app.circleci.com/pipelines/github/edx-solutions/edx-platform/527/workflows/d347c3c8-8e78-47cc-a639-99fe59cf0cac/jobs/5483/parallel-runs/1?filterBy=FAILED

Comment (ihtram) [reply openedx/core/djangoapps/cors_csrf/tests/test_middleware.py:274]: @moeez96 the link you shared has the different code at that line. Please compare that line with the one I shared above and change that line accordingly.

**Pull request merged**